buildspec.yml

```diff
@@ -14,9 +14,10 @@ env:
 phases:
   install:
     runtime-versions:
-      python: 3.7
+      python: 3.9
     commands:
       - echo "Installing necessary dependencies"
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
```

Comment:
Add a comment why this has been added, in theorie this should be fixed by AWS Codebuild base image (managed by AWS) and could be removed here in future.

Reply:
done